.github/workflows/tekton_bundle_push.yaml

```diff
@@ -0,0 +1,127 @@
+---
```

Comment:
I wouldn't expect to see this file in a PR for adding a yaml linter. This should be in a different PR.

Reply:
On a related note, this PR has 20 commits. Is that expected? Maybe it should be squashed down a bit?

Reply:
> I wouldn't expect to see this file in a PR for adding a yaml linter. This should be in a different PR.

The PR adds configuration for where to find the ".yamllint" config file, so in my mind it's fair game to also include a config. Also I don't think it's fair to say the inclusion of the config file is somehow overloading this PR.

Reply:
My comment is not in the config file. It is in the Tekton Bundle Push workflow which should not be part of a PR to add yaml linting.

Regarding the rebase, those commits shouldn't be here if they were already in main. Please, clean up the PR and squash the commits so we can review properly.